src/templates/hi/accessibility_statement.html

```diff
@@ -28,7 +28,7 @@ <h2 id="overview"><a href="#overview" class="anchor-link">अवलोकन</a>
           Web Almanac सभी लोगों के लिए डिजिटल पहुंच सुनिश्चित करने के लिए प्रतिबद्ध है। हम सभी के लिए यूजर एक्सपीरियंस में लगातार सुधार कर रहे हैं और जरूरी अभिगम्यता मानकों को लागू कर रहे हैं।
         </p>
         <p>
-          इस कथन की अंतिम समीक्षा <time datetime="2020-06-02">2nd June 2020</time> को की गई थी।
+          इस कथन की अंतिम समीक्षा <time datetime="2021-12-01">1st December 2021</time> को की गई थी।
```

Comment:
Should this be translated too? something like this?

```suggestion
          इस कथन की अंतिम समीक्षा <time datetime="2021-12-01">1 दिसंबर 2021</time> को की गई थी।
```

Also, to get listed on our contributors page, could you add a profile of yourself to `src/config/2021.json` in the `translators` team. Please list yourself in alphabetically order by first name so maybe around line 930? You can add a website, github profile (use this as your id), linkedin and/or twitter handle.

Reply:
my apologies for this little mistake and now it's almost done

Reply:
Well you copied what was there before so not really your fault. But happened to notice when reviewing this.

Reply:
> Well you copied what was there before so not really your fault. But happened to notice when reviewing this.

ok, it's my pleasure.

Reply:
So is the above the current translation? Can I commit this?